rustls/src/server/server_conn.rs

```diff
@@ -799,6 +799,16 @@ mod connection {
                 sig_schemes,
             }))
         }
+
+        /// Yields an iterator over the buffered chunks of bytes
+        ///
+        /// Will yield an empty iterator after `accept()` has returned `Ok(Some(_))` or `Err(_)`.
```

Comment:
I think this is returning a slice or an empty slice in the current impl, not an iterator. Does this bit of docs need updating?

Reply:
Yup, but let's first decide if we think this API is useful.